src/foundry/client/core/keybindings.d.mts

```diff
@@ -172,15 +174,14 @@ declare global {
      * @param context - The context data of the event
      * @internal
      */
-    protected static _onDismiss(context?: KeyboardEventContext): boolean;
+    protected static _onDismiss(context?: KeyboardEventContext): Promise<boolean>;
 
     /**
      * Open Character sheet for current token or controlled actor
-     * @param event   -
      * @param context - The context data of the event
      * @internal
      */
-    protected static _onToggleCharacterSheet(event?: KeyboardEvent, context?: KeyboardEventContext): ActorSheet | null;
+    protected static _onToggleCharacterSheet(context?: KeyboardEventContext): ActorSheet | null;
```

Comment:
Should the return type maybe become broader, e.g. ActorSheetV2?

Reply:
it should be whatever `game.toggleCharacterSheet()` returns.  The jsdoc for that still shows `ActorSheet|null` but perhaps that's not right?  I'll admit to not understanding the whole Application/Document/Sheet thing in any depth.

Reply:
Oh honestly might be worth it to just tag as `ReturnType<Game["toggleCharacterSheet"]>` then